code/index.html

```diff
@@ -1,30 +1,92 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
   <head>
     <meta charset="utf-8" />
+    <link rel="stylesheet" href="style.css">
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Project 3</title>
-    <!-- dont forget to add a css file and link it here! -->
+    <link rel="shortcut icon" href="img/cupicon.jpg" type="image/x-icon">
+    <link href="https://fonts.googleapis.com/css?family=Bowlby+One+SC&display=swap" rel="stylesheet">
+    <!-- <link href="https://fonts.googleapis.com/css?family=Shadows+Into+Light&display=swap" rel="stylesheet"> -->
+    <link href="https://fonts.googleapis.com/css?family=Nanum+Myeongjo&display=swap" rel="stylesheet">
+    <title>[BUNS] — Made to Order</title>
+ 
+ 
   </head>
   <body>
-    <!-- video or image as a header is cool :) -->
+    <header class="title">
+      <div class="title-text">
+        <h1>[BUNS]</h1></div>
+    </header>
+
 
     <!-- Signup form -->
+  <main class="main">
+    <div class="form">
+      <form action="https://httpbin.org/anything" method="POST">
+
+        <label class="textArea">Your Name:<br>
+        <input type="text" required name="fullname" placeholder="Full Name"><br></label>
```

Comment:
The `<br>` element is from start a breakpoint being used in text purpose, not for styling purpose. This styling could be fixed with only css, so maybe next time try out creating this styling with only css!